patches/server/0848-Folia-scheduler-and-owned-region-API.patch

```diff
@@ -363,15 +363,12 @@ index 0000000000000000000000000000000000000000..374abffb9f1ce1a308822aed13038e77
 +        Validate.notNull(plugin, "Plugin may not be null");
 +        Validate.notNull(task, "Task may not be null");
 +        Validate.notNull(unit, "Time unit may not be null");
-+        if (delay < 0L) {
-+            throw new IllegalArgumentException("Delay may not be < 0");
-+        }
 +
 +        if (!plugin.isEnabled()) {
 +            throw new IllegalPluginAccessException("Plugin attempted to register task while disabled");
 +        }
 +
-+        return this.scheduleTimerTask(plugin, task, delay, -1L, unit);
++        return this.scheduleTimerTask(plugin, task, Math.max(1L, delay), -1L, unit);
```

Comment:
This changes behavior now? Since I cant set delay to 0 anymore, it'll be set to 1.

Reply:
You can not set to 0, 1 is minimum required